content/en/storage-providers/setup/prerequisites.md

```diff
@@ -0,0 +1,157 @@
+
+```shell
+export BELLMAN_CUSTOM_GPU="NVIDIA GeForce RTX 3090:10496"
+```
+
+Nvidia RTX 3090 was used in this example. Remember to edit it with your GPU and amount of Cuda cores.
+
+### Configure parameters location
+
+For the miner to start, it will need to read and verify the Filecoin proof parameters. These can be downloaded in advance (recommended), or otherwise the init process will. Proof parameters consist of several files, which in the case of 32 GiB sectors, total **over 100 GiB**.
+
+We recommend setting a custom location to store parameters and proofs parent cache -created during the first run- with:
+
+```sh
+export FIL_PROOFS_PARAMETER_CACHE=/path/to/folder/in/fast/disk
+export FIL_PROOFS_PARENT_CACHE=/path/to/folder/in/fast/disk2
+```
+
+Parameters are read on every (re)start, so using disks with very fast access, like NVMe drives, will speed up miners and workers (re)boots. When the above variables are not set, things will end up in `/var/tmp/` by default, which **often lacks enough space**.
+
+Parameters will be downloaded automatically when the miner is initiated. You can also [optionally download]({{< relref "#downloading-parameters" >}}) them before initializing.
+
+### Creating wallets for the miner
+
+You will need at least a BLS wallet (`f3...` for mainnet) for mining. We recommend using [separate owner and worker addresses]({{< relref "addresses" >}}):
+
+```sh
+# A new BLS address to use as owner address:
+lotus wallet new bls
+f3...
+
+# A new BLS address to use as worker address:
+lotus wallet new bls
+f3...
+```
+
+{{< alert icon="callout" >}}
+Next make sure to [send some funds]({{< relref "manage-fil" >}}) to the **worker address** so that the miner setup can be completed. The amount you should initialize with varies with gas fees, but 0.1 FIL is generally a safe amount. The sender doesn't have to be any particular address and can be specified using the `from` flag. If `from` is unspecified, the sender will default to the `owner` address, in which case the `onwer` must have the 0.1 FIL. If the `owner` is also unspecified, the wallet's default address is used as the owner and that address must have the 0.1 FIL. 
+{{< /alert >}}
+
+For additional information about the different wallets that a miner can use and how to configure them, read the [miner addresses guide]({{< relref "addresses" >}}).
+
+{{< alert icon="tip" >}}
+Safely [backup your wallets]({{< relref "manage-fil#exporting-and-importing-addresses" >}})!
+{{< /alert >}}
+
+
+## Optional prerequisites
+
+These prerequisites are optional and can be used on case by case basis. Please make sure to understand the use case before performing these steps.
+
+### Install extra dependencies
+
+This step is only if you are running an Nvidia GPU and would prefer to use OpenCL instead of CUDA. Please note that Cuda is after than OpenCL for sealing and proving workload. Most Linux distributions contain this package in their package manager:
```

Comment:
```suggestion
This step is only necessary if you are running an Nvidia GPU and would prefer to use OpenCL instead of CUDA. Please note that Cuda is recommended over OpenCL for sealing and proving workload. Most Linux distributions contain this package in their package manager:
```